packages/atomic/src/components/search/facets/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -356,7 +365,7 @@ export class AtomicRatingFacet
   }
 
   private get valuesToRender() {
-    return this.facetState.values.filter(
+    return this.facet.state.values.filter(
```

Comment:
Helps with race condition between update of facetState object and the isHidden() call.

Reply:
The `render` function is triggered whenever a `@State` is updated, therefore I would expect it to only render the `@State`, not a value that was dynamically updated.

This function is used during rendering, which makes sense IMO since it's suffixed with `ToRender`.

I think we should find other means of avoiding the race condition.